### DISCUSSION TITLE
Porter binary is duplicated by install script

### DISCUSSION THREAD

**Discussion by tamirkamara** [getporter/porter#2014] [Q&A]
Hi,
I was wondering why the published [install script for linux](https://cdn.porter.sh/latest/install-linux.sh) is saving the porter binary twice: 
```bash
curl -fsSLo $PORTER_HOME/porter $PORTER_MIRROR/$PORTER_PERMALINK/porter-linux-amd64
chmod +x $PORTER_HOME/porter
cp $PORTER_HOME/porter $PORTER_HOME/runtimes/porter-runtime
```

I tried replacing the cp step with a symlink and it seems to work...

**Comment by carolynvs** [comment]
At the moment, the runtime binary is the same as the client binary on linux only. That will change in the future.

I haven't tried using a symlink. When you run porter build, and then install that bundle, did it work for you? We can change it to a symlink if that works.

**Reply by tamirkamara** [reply]
yes, it does work for me. I believe you also do something similar in one of the docker images here: https://github.com/getporter/porter/blob/main/build/images/client/Dockerfile#L8

I can open the PR if we agree to do this...

**Reply by carolynvs** [reply]
Yes, thanks that would be helpful! I appreciate your testing out the fix.
